polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -872,43 +767,25 @@ function IndexTableBase({
       <div className={styles.EmptySearchResultWrapper}>{emptyStateMarkup}</div>
     );
 
-  const tableWrapperClassNames = classNames(
-    styles.IndexTableWrapper,
-    Boolean(selectAllActionsMarkup) &&
-      selectMode &&
-      !pagination &&
-      styles.IndexTableWrapperWithSelectAllActions,
-  );
+  const tableWrapperClassNames = classNames(styles.IndexTableWrapper);
 
-  const paginationWrapperClassNames = classNames(
-    styles.PaginationWrapper,
-    shouldShowBulkActions && styles.PaginationWrapperWithSelectAllActions,
-    isScrolledPastTop && styles.PaginationWrapperScrolledPastTop,
-  );
+  const paginationWrapperClassNames = classNames(styles.PaginationWrapper);
 
   const paginationMarkup = pagination ? (
-    <div
-      className={paginationWrapperClassNames}
-      style={
-        {
-          '--pc-index-table-pagination-top-offset': `${selectAllActionsPastTopOffset}px`,
-        } as React.CSSProperties
-      }
-    >
+    <div className={paginationWrapperClassNames}>
       <Pagination type="table" {...pagination} />
     </div>
   ) : null;
 
   return (
     <>
       <div className={styles.IndexTable}>
-        <div className={tableWrapperClassNames} ref={tableMeasurerRef}>
-          {!shouldShowBulkActions && !condensed && loadingMarkup}
+        <div className={tableWrapperClassNames}>
```

Comment:
```suggestion
        <div className={styles.IndexTableWrapper}>
```

Light suggestion to use class name directly since multiple class names aren't being combined. Would require removing the `tableWrapperClassNames` variable.